dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/AndroidTracerTest.kt

```diff
@@ -568,6 +569,35 @@ internal class AndroidTracerTest {
         assertThat(traceIdSpan1).isNotEqualTo(traceIdSpan2)
     }
 
+    @Test
+    fun `M set correct propagating style W setting tracing header type`(forge: Forge) {
+        // Given
+        val threshold = forge.anInt(max = 100)
+        val tracingHeaderStyle = forge.anElementFrom(
+            setOf(
+                TracingHeaderType.DATADOG,
+                TracingHeaderType.B3,
+                TracingHeaderType.B3MULTI,
+                TracingHeaderType.TRACECONTEXT
+            )
+        )
+        // When
+        val tracer = testedTracerBuilder
+            .setServiceName(fakeServiceName)
+            .setPartialFlushThreshold(threshold)
+            .setTracingHeaderTypes(setOf(tracingHeaderStyle))
```

Comment:
ideally we need to inject multiple types, to check that in end we have an expected format (types joined into a single string), although in this case test code will be quite similar to the code under test.